packages/lib/payment/handlePayment.ts

```diff
@@ -58,9 +60,12 @@ const handlePayment = async (
         currency: selectedEventType?.metadata?.apps?.[paymentAppCredentials.appId].currency,
       },
       booking.id,
+      booking.userId,
+      booking.user?.username ?? null,
+      bookerName,
       bookerEmail,
       paymentOption,
-      evt.title
+      selectedEventType.title
```

Comment:
Previous PR added the event title (eg "30 minute meeting with Roald Dahl") instead of the event type title (eg "30 minute meeting"). 

Because I'm looking to use multiple accounts on the same Stripe account, it would be more useful to be able to aggregate on the event type title to show earninings per event type.

If it's also useful to have the calendar event title, I can add it back in addition.

Reply:
Yeah, let's keep both. BookingTitle and eventTitle. What you think?